distributed/client.py

```diff
@@ -1507,7 +1507,12 @@ def __del__(self):
         # If the loop never got assigned, we failed early in the constructor,
         # nothing to do
         if self.__loop is not None:
-            self.close()
+            try:
+                asyncio.get_running_loop()
+            except RuntimeError:
+                pass
```

Comment:
when using the `asynchronous=False` Client (the default) there's unlikely to be a running loop so .close() won't be called